dg/lib/cloud_pak_for_data/cpd_manager.py

```diff
@@ -73,6 +75,11 @@ def download_cpd_installer(self):
 
         pass
 
+    def execute_cloud_pak_for_data_installer(self, args) -> subprocess.CompletedProcess:
```

Comment:
Intead of importing subprocess and dg.utills.process we should mabye only import dg.utils.process and "inherit" the stuff from subprocess in dg.utils.process ? Like this `subprocess.CompletedProcess` 

What do you think ?

Reply:
What do you mean exactly? Do you mean adding 

```
from subprocess import CompletedProcess
```

to `dg.utils.process` and then use the following method signature in `dg.lib.cloud_pak_for_data.cpd_manager`?

```
def execute_cloud_pak_for_data_installer(
    self, args
) -> dg.utils.process.CompletedProcess:
```

While this would work, I am not convinced yet as it is not clear that `dg.utils.process.CompletedProcess` is just another name for `subprocess.CompletedProcess` and not a new type.